storage/src/nodestore/persist.rs

```diff
@@ -245,7 +276,7 @@ impl NodeStore<Committed, FileBacked> {
     }
 }
 
-impl NodeStore<Committed, FileBacked> {
+impl<S: WritableStorage + 'static> NodeStore<Committed, S> {
```

Comment:
Question only: Why was 'static required here? I don't think it matters because S will never be a reference, but it seems like it shouldn't be required.